Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -57,7 +61,6 @@ public class OpeningBreakoutAlgorithm : QCAlgorithm
         private const string Symbol = "SPY";
 
         // plotting and logging control
-        private const bool EnablePlotting = true;
         private const bool EnableOrderUpdateLogging = false;
```

Comment:
You need to touch that, otherwise you have two flags (one const + one preprocessor directive) :)

As @jaredbroad doesn't like preprocessor directives, in order to remove the build warnings we'll have to leave the bool constants and add `#pragma warning` directives instead (removing the `#define` statements you added)